site/content/docs/guides/zapping-the-top-10.md

```diff
@@ -1,10 +1,16 @@
 ---
 title: "ZAPping the OWASP Top 10"
+description: "This document gives an overview of the automatic and manual components provided by the OWASP Zed Attack Proxy (ZAP) that are recommended for testing each of the OWASP Top Ten Project 2017 risks."
+images:
+- https://raw.githubusercontent.com/OWASP/Top10/master/2017/images/OWASP-Top-10.png
+tags: 
+- top10
+- guide
 type: page
 date: "2020-01-30"
 ---
 
-This document gives an overview of the automatic and manual components provided by the OWASP Zed Attack Proxy Project (ZAP) that are recommended for testing each of the OWASP Top Ten Project 2017 risks.
+This document gives an overview of the automatic and manual components provided by the OWASP Zed Attack Proxy (ZAP) that are recommended for testing each of the OWASP Top Ten Project 2017 risks.
```

Comment:
Either "the OWASP Zed Attack Proxy Project" (as before) or "OWASP Zed Attack Proxy Project" (no "the" or "project").

Reply:
Still think the "the" doesnt sound right :/ "provided by OWASP Zed Attack Proxy" sounds better to me, but not a biggie...

Reply:
Missed it sorry, will tweak shortly